javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/model/typesystem/ReferenceTypeTest.java

```diff
@@ -246,10 +259,10 @@ void testIsAssignableByGenericsInheritance() {
 
     @Test
     void testGetAllAncestorsConsideringTypeParameters() {
-        assertTrue(linkedListOfString.getAllAncestors().contains(object));
-        assertTrue(linkedListOfString.getAllAncestors().contains(listOfStrings));
-        assertTrue(linkedListOfString.getAllAncestors().contains(collectionOfString));
-        assertFalse(linkedListOfString.getAllAncestors().contains(listOfA));
+        assertThat(linkedListOfString.getAllAncestors(), hasItem(object));
```

Comment:
Converted to the hamcrest matchers here as it gives a more meaningful error message without needing to manually add in a message string (i.e. that a value is missing, as opposed to the junit approach just informs you that a true value was actually false, and vice-versa)